galloc/docs/optimisation.md

```diff
@@ -0,0 +1,315 @@
+# Optimisation of galloc
+We've tried to optimise galloc in some ways, but none of them showed significant performance improvement, so we'd **strongly suggest to keep the current implementation.**
+
+The current implementation uses static buffer in addition to standard <kbd>dlmalloc</kbd>-like implementation.
+
+![Total perfomance comparison](./images/total-performance-comparison.png)
```

Comment:
Most of this work involved investigation and research, with some portions being done manually. However, I am uncertain about the necessity of uploading the scripts (since it's not the complete portion of the work), as well as generating charts in CI, given that the allocator is not frequently updated. What are your thoughts?